lib/bundler.rb

```diff
@@ -195,6 +195,16 @@ def tmp(name = Process.pid.to_s)
 
     def rm_rf(path)
       FileUtils.remove_entry_secure(path) if path && File.exist?(path)
+    rescue ArgumentError => e
+      if /world writeable/ =~ e.message
```

Comment:
Actually, when I searched google for this, Nanoc changed remove_entry_secure to rm_rf (See https://github.com/nanoc/nanoc/commit/aeb71156d6a97851f9f262ae68eb09074432b314 and https://github.com/nanoc/nanoc/issues/465).  
But since we're using remove_entry_secure, I think it's better to throw the error and warn our user about the global-writable issue.